code/modules/wiremod/shell/brain_computer_interface.dm

```diff
@@ -186,8 +182,8 @@
 	charge_action = new(src)
 	bci.actions += list(charge_action)
 
-	RegisterSignal(shell, COMSIG_CARBON_GAIN_ORGAN, PROC_REF(on_organ_implanted))
-	RegisterSignal(shell, COMSIG_CARBON_LOSE_ORGAN, PROC_REF(on_organ_removed))
+	RegisterSignal(shell, COMSIG_ORGAN_IMPLANTED, PROC_REF(on_organ_implanted))
+	RegisterSignal(shell, COMSIG_ORGAN_REMOVED, PROC_REF(on_organ_removed))
```

Comment:
I this fixes an issue in local testing where BCIs just flat out do not let you use the "Mob say" functionality (this part was not actually ported correctly from /tg/)